code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -40,21 +40,29 @@
 	if(istype(H))
 		var/obj/item/organ/external/affected = H.get_organ(user.zone_sel.selecting)
 		if(!affected)
-			user << "<span class='danger'>\The [H] is missing that limb!</span>"
+			to_chat(user, "<span class='danger'>\The [H] is missing that limb!</span>")
 			return
 		else if(affected.robotic >= ORGAN_ROBOT)
-			user << "<span class='danger'>You cannot inject a robotic limb.</span>"
+			to_chat(user, "<span class='danger'>You cannot inject a robotic limb.</span>")
 			return
 
+		if(!H.stat)
+			if(H != user)
+				if(H.a_intent != I_HELP)
+					to_chat(user, "<span class='notice'>[H] is resisting your attempt to inject them with \the [src].</span>")

```

Comment:
This is still going to go "hey, they're resisting!" as soon as you try and inject them. Possibly untrue (what if they're just not on help intent, and not actively trying to resist?), and possibly spammable on both ends - click someone a bunch and then move away before the 3 seconds are up.